CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -799,31 +880,313 @@ private static void OnTimeoutEvent(object source, ElapsedEventArgs e)
             timer.Stop();
             timer.Dispose();
         }
+    }
+    /// <summary>
+    /// Manages the pot in a Texas Hold'em game, including adding and distributing chips.
+    /// </summary>
+    public class TexasHoldEmPot
```

Comment:
Is the idea that we incorporate a pot in each game? We'd copy and paste this code into each game's manager? Wouldn't it be easier to separate this into it's own module and import it into each game? It's the same question I had for the player manager in the other PR.

Other than that, this looks good

Reply:
Imo since this file is already >800 lines atp there's no reason not to at least move it to it's own file

Reply:
Gonna resolve though for the sake of time